superset-frontend/src/components/ImportModal/index.tsx

```diff
@@ -68,11 +145,11 @@ const ImportChartModal: FunctionComponent<ImportChartModalProps> = ({
   const {
     state: { passwordsNeeded },
     importResource,
-  } = useImportResource<ChartObject>('chart', t('chart'), handleErrorMsg);
+  } = useImportResource<any>(resourceName, resourceLabel, handleErrorMsg);
```

Comment:
do you want to constrain this with a set of type arguments for each type of model? You can do the same for the resourceName.. not a blocker, though, just if you want to make the types stronger

Reply:
You mean using an union of all types instead of `any`?

Reply:
+1 with elizabeth, where you union all the types allowed to be imported.

@eschutho that's what you meant right

Reply:
yeah, that's what I was thinking.

Reply:
We actually don't need this, I removed it in https://github.com/apache/incubator-superset/pull/12003 to avoid having to rebuild this PR.